src/index.html

```diff
@@ -13,6 +13,52 @@
     />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="card"
+      data-qa="card"
+    >
+      <div class="card__top">
+        <img
+          src="images/imac.jpeg"
+          class="card__top-img"
+          alt="Apptle Monoblock"
+        />
+      </div>
+
+      <div class="card__botoom">
+        <h2 class="card__botoom-tittle">
+          APPLE A1419 iMac 27" Retina
+          <br />
+          5K Monoblock (MNED2UA/A)
+        </h2>
+
+        <p class="card__botoom-code">Product code: 195434</p>
+
+        <div class="card__botoom-stars">
+          <div class="stars stars--4">
+            <div class="stars__item"></div>
+            <div class="stars__item"></div>
+            <div class="stars__item"></div>
+            <div class="stars__item"></div>
+            <div class="stars__item"></div>
+          </div>
+
+          <p class="card__botoom-text">Reviews: 5</p>
+        </div>
+
+        <div class="card__botoom-cost">
+          <p class="card__botoom-price">Price:</p>
+          <p class="card__botoom-sum">$2,199</p>
+        </div>
+
+        <a
+          class="card__botoom-button"
```

Comment:
There is a typo in the class name `card__botoom-button`. It should be `card__bottom-button` to maintain consistency and avoid potential styling issues.